app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,15 @@
+class LikesController < ApplicationController
+  def create
+    @like = Like.new(
+      author_id: params[:user_id],
+      post_id: params[:post_id]
+    )
```

Comment:
- [x] Let's ensure that a like is created on behalf of the currently logged in user (current_user) of the app.

```suggestion
  def create
    @like = Like.new(
      author_id: current_user.id,
      post_id: params[:post_id]
    )
```

Reply:
This issue is well resolved! ✔️